src/CalculatorUnitTests/DateCalculatorUnitTests.cpp

```diff
@@ -390,7 +396,7 @@ TEST_METHOD(TestSubtraction)
     //    DateTime endDate;
 
     //    // Subtract Duration
-    //    bool isValid = m_DateCalcEngine.SubtractDuration(DateUtils::SystemTimeToDateTime(datetimeSubtractCase[testIndex].startDate),
+    //    bool isValid = m_DateCalcEngine->SubtractDuration(DateUtils::SystemTimeToDateTime(datetimeSubtractCase[testIndex].startDate),
```

Comment:
nit: SubtractDuration no longer returns a bool

(It's okay with me if you won't want to fix these commented-out tests right now.)